WordPress/src/main/java/org/wordpress/android/ui/stats/models/InsightsAllTimeModel.java

```diff
@@ -0,0 +1,65 @@
+package org.wordpress.android.ui.stats.models;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.Serializable;
+
+public class InsightsAllTimeModel implements Serializable {
```

Comment:
For consistency with our other models, maybe kill the `Model` text at the end of the name?